x-pack/plugin/eql/src/main/java/org/elasticsearch/xpack/eql/expression/function/scalar/whitelist/InternalEqlScriptUtils.java

```diff
@@ -21,4 +22,8 @@
     public static String substring(String s, Number start, Number end) {
         return (String) SubstringFunctionProcessor.doProcess(s, start, end);
     }
+
+    public static String between(String s, String left, String right, Boolean greedy, Boolean caseSensitive) {
```

Comment:
Can you move this one before `substring` (to follow the alphabetical order), please?